Docs/Configuration.tex

```diff
@@ -2765,126 +2762,129 @@ \section{Misc}\label{misc}
 
 \subsection{Introduction}\label{miscintro}
 
-This section contains miscellaneous configuration affecting OpenCore operating system loading behaviour
-as well as other entries, which do not go to any other section.
+This section contains miscellaneous configuration options that do not readily fit into other sections.
+These options affect the loading behaviour of the OpenCore operating system in addition to other items.
 
-OpenCore tries to follow ``bless'' model also known as ``Apple Boot Policy''. The primary specialty of
-``bless'' model is to allow embedding boot options within the file system (and be accessible through a
-specialised driver) as well as supporting a broader range of predefined boot paths compared to the
-removable media list found in the UEFI specification.
+OpenCore broadly follows a ``bless'' model also known as the ``Apple Boot Policy''. The primary purpose of
```

Comment:
It should be '**the** bless model' at every occurrence, I think (it's Apple specific).